_includes/v22.1/ui/sessions.md

```diff
@@ -49,7 +49,7 @@ The **Cancel session** button ends the session. The client that holds this sessi
   - **Application name** {% include_cached new-in.html version="v22.1" %} shows the name of the application connected to the session.
```

Comment:
Does this "name" need to be capitalized as well? Just checking.

Reply:
Yes. Thanks for the catch.